### ISSUE TITLE
Documentation Loop

### ISSUE THREAD

**Issue by dustinaja** [harvesthq/chosen#2662]
Going in a loop, documentation is unclear.

### Steps to reproduce

Tell us how to reproduce this issue.

  1. Google for "jQuery Dropdown Plugin"
  2. Click the first link for https://plugins.jquery.com/tag/dropdown/
  3. Click the first link for https://plugins.jquery.com/chosen/
  4. Click the download link for https://github.com/harvesthq/chosen/archive/v1.6.2.tar.gz
  5. Extract the files.
  6. Search for jquery.chosen-1.6.2.js
  7. After searching all folders, scratch head.
  8. Return to project root https://github.com/harvesthq/chosen
  9. Click downloads https://github.com/harvesthq/chosen/releases/
  10. Observe we're back at a similar page to step 3.
  11. Return to project root https://github.com/harvesthq/chosen
  12. Try documentation https://harvesthq.github.io/chosen/
  13. Ah ha! Click Download. https://github.com/harvesthq/chosen/releases
  14. Hmm... another loop.
  15. Return to documentation https://harvesthq.github.io/chosen/
  16. Scroll to bottom.
  17. Yes! Finally. Simple instructions, "Using Chosen is easy as can be." Exactly what I need. Proceed to step 1 of Setup.
  18. Download the plugin... https://github.com/harvesthq/chosen/releases
  19. Ugh, looks like another loop.
  20. Attempt to complete step 1 anyways, "Download the plugin and copy the chosen files to your app."
  21. Download the plugin again, I must have missed something.
  22. Activate the plugin on the select boxes of your choice: $(".chosen-select").chosen()
  23. Huh... there ARE no chosen files to copy to my app.

### Bonus Steps

  24. Bonus step, begin to look at issues. Find an issue where someone says, "Chosen not including the compiled files is well discussed and intentional."
  25. Scratch head, look at README.md again for "well discussed" information about not including the chosen files that were mentioned in step 20.

### Expected behavior

Download package. Extract package. Copy jquery.chosen.js to project. Copy jquery.chosen.css to project. Begin work.

### Actual behavior

Expected behavior worked for steps 1 through 5. Step 6 diverges from expected behavior.

### Environment

  - **1.6.1**:

  - **jQuery**:

  - **Chrome**:

  - **Centos**:

### Additional information

While this may be a bit satirical, I still don't actually know the quickest way to "download" this program and add it to my project. I don't use bower. I think I'm going to copy it from the CDN.

**Comment by koenpunt** [comment]
As the jQuery plugins site is being retired, we've removed support for it. https://github.com/harvesthq/chosen/pull/2550

**Comment by koenpunt** [comment]
> While this may be a bit satirical, I still don't actually know the quickest way to "download" this program and add it to my project. I don't use bower. I think I'm going to copy it from the CDN.

You can download the latest release zip from the [releases page](https://github.com/harvesthq/chosen/releases), which is `chosen_v1.6.2.zip`

**Comment by dustinaja** [comment]
Assume jQuery plugins site doesn't exist.

I still expect to be able to visit github, and download javascript and css files quickly.

The linked issue is great. Including npm support is great. I use npm for a couple projects personally, but this doesn't resolve the issue where plain ol' javascript files are not readily available.

**Comment by koenpunt** [comment]
I've created the 1.6.2 release yesterday, but forgot to add the zip with the prebuilt files, so that could've been the source of your confusion.

**Comment by dustinaja** [comment]
Thanks, that's the answer.

I'm befuddled, each time I went to that page, I only remember seeing two entries.

 - Source code (zip)
 - Source code (tar.gz)

Every time I looked I didn't notice the `chosen_v1.6.2.zip` entry that you just pointed out. I see now it's clearly there, but I didn't see it.

Ahh, just saw your comment pop up while I was writing this. Phew.

Yes, just tested download, this is exactly what I was looking for and expected to find. Thanks.
